projects/plugins/jetpack/changelog/fix-font-name-undefined

```diff
@@ -0,0 +1,4 @@
+Significance: patch
```

Comment:
I'm assuming that a version bump is not required on this occasion. CMIIW 🙏 
https://github.com/Automattic/jetpack/blob/e2f6592c65e7da53f7d4010da2a19b4443145567/docs/monorepo.md?plain=1#L374